Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityEditorSettings.cs

```diff
@@ -144,7 +144,38 @@ private static bool IsNewSession
             }
         }
 
-        private static bool FindDirectory(string directoryPathToSearch, string directoryName, out string path)
+        /// <summary>
+        /// Finds the path of a directory relative to the project folder.
+        /// </summary>
+        /// <param name="directoryPathToSearch">
+        /// The subtree's root path to search in.
+        /// </param>
+        /// <param name="directoryName">
+        /// The name of the directory to search for.
+        /// </param>
+        public static bool FindRelativeDirectory(string directoryPathToSearch, string directoryName, out string path)
```

Comment:
This wasn't intended to be public. It likely belongs in a utility class.

Reply:
I agree. Would `EditorProjectUtilities` be a good place for this?

Reply:
that seems like a reasonable spot